src/api.ts

```diff
@@ -12,5 +12,11 @@ export function getPeople(): Promise<Person[]> {
   // keep this delay for testing purpose
   return wait(500)
     .then(() => fetch(API_URL))
-    .then(response => response.json());
+    .then(response => {
+      if (!response.ok) {
+        throw new Error();
```

Comment:
Consider providing a more descriptive error message in the `Error` constructor. This will make it easier to debug issues related to failed API requests. For example, you could include the status code or a custom message like `Failed to fetch people data`.